### ISSUE TITLE
Remove Nuxt

### ISSUE THREAD

**Issue by dlercier** [makinacorpus/Recolto#12]

**Comment by cbaccam** [comment]
@SimonMellerin @dlercier I would be interested to take this issue.

**Comment by SimonMellerin** [comment]
@cbaccam thanks!

May be we can make this issue step by step. 

I have quickly checked and here is a list of component from Nuxt/ui to remove from the project:

- [ ] UIcon
- [ ] UButton
- [ ] USelectMenu
- [ ] URadio
- [ ] UToggle
- [ ] UInput
- [ ] UPopover
- [ ] USkeleton

**I propose to make one PR per component to be replaced.**

If some components are too complicated to develop, you could consider replace nuxt/ui with another components library.

**Comment by cbaccam** [comment]
ok for one PR per component. I will start after the holidays to give me time to learn properly vue.js . I am more familiar with javascript/typescript with angular/jquery.

when you mean vanilla js,  is it  the standard html 5 code ?

**Comment by SimonMellerin** [comment]
> when you mean vanilla js, is it the standard html 5 code ?

Yes that was what I was thinking about. But, maybe it won't be so easy to do. And I don't think it's worth spending too much time on it.

We usually use the primevue component library on our Vuejs projects. We could probably use it on recolto too.
